lib/core/utils.js

```diff
@@ -87,6 +87,13 @@ function maxWireVersion(topologyOrServer) {
     return topologyOrServer.ismaster.maxWireVersion;
   }
 
+  if (typeof topologyOrServer.lastIsMaster === 'function') {
```

Comment:
What `topologyOrServer` does not have the `lastIsMaster` function?

Reply:
Technically `lib/core/sdam/server.js` does not have it. I don't think it is ever actually used in this function, but I wanted to be safe.